src/classes/HH_Container_LCTRL.cls

```diff
@@ -37,6 +37,28 @@
 public class HH_Container_LCTRL {
     
     /*******************************************************************************************************
+    * @description trick the packaging spider to pickup references to custom labels only used by our
+    * lightning components.
+    */
+    public static void packageLabels() {
```

Comment:
This looks like it only references labels that you created anew for this work?  Do you think it should contain references to ALL labels that you use?

The reason it might be better to do that is there is no guarantee that those labels are or will always be referenced elsewhere.  Other completely unrelated code could change or be removed and that would break this.